core/src/main/resources/reference.conf

```diff
@@ -169,6 +169,12 @@ pekko.persistence.r2dbc {
     # Enabling this has some performance overhead.
     # A fast query for Postgres is "SELECT 1"
     validation-query = ""
+
+    # FQCN of an OptionsCustomizer. If non-empty, it must be the fully qualified class
+    # name of a class implementing the trait ConnectionFactoryProvider.OptionsCustomizer.
+    # The class must have a constructor with a single parameter of type ActorSystem[_].
+    # If this setting is empty, the default no-op customizer will be used.
+    options-customizer = ""
```

Comment:
this name is too short, afiac - when I see Option, I think scala.Option. Can we rename this as `connectionfactoryoptions-customizer`? Likewise for the actual classes (ie include ConnectionFactory in the class names).